Automattic/wp-calypso
# Marketplace: Add footer with static info to plugin details

#### Proposed Changes

* Add the same footer existing in  https://wordpress.com/plugins to plugin details page

|Before | After|
|-------|------|
|  ![CleanShot 2022-09-16 at 11 02 24@2x](https://user-images.githubusercontent.com/3519124/190600273-a081801a-4b52-4097-a0f2-8910690e24c9.png)       |![CleanShot 2022-09-22 at 11 52 19@2x](https://user-images.githubusercontent.com/3519124/191716814-a61d0048-d7a0-40b1-8862-8adad58801f7.png)|

For reference, the previous style was: [Initial before screenshot](https://user-images.githubusercontent.com/3519124/190599962-9984690a-3a37-4bd7-9e5b-55c00c162640.png)

#### Testing Instructions



* Navigate to a plugin details page, e.g. `/plugins/wordpress-seo-premium` and make sure the added footer exists and it matches the expected style

~@vinimotaa Could you please confirm if the following differences with the Figma design of the footer are expected or they should be changed?~
~- The main title reads `WordPress.com is the best place to get your plugins.` in the design but the current footer contains the text `Add additional features to your WordPress site. Risk free.`~
~- The paragraphs titles are slightly different in the design than in the current implementation~
~- The font-size of the 3 paragraphs are 14px in the design but 16px in the current footer~
~- There is a 21px margin at the bottom of the paragraphs that do not exist in the design~

~Please note that the content of the paragraphs is a _lorem ipsum_ so I assume the existing ones are correct~

**Note:** The above questions ⬆️  are already being handled as part of:
The more generic task 
- https://github.com/Automattic/wp-calypso/issues/65574

And most of the design updates have been done as part of:
- https://github.com/Automattic/wp-calypso/pull/67636

#### Pre-merge Checklist






Fixes #67663
